products/workers/src/content/runtime-apis/durable-objects.md

```diff
@@ -421,5 +421,5 @@ Any uncaught exceptions thrown by the Durable Object's `fetch()` handler are pro
 
 The Cloudflare API supports retrieving a list of Durable Objects within a namespace and a list of namespaces associated with an account.
 
-The API for listing namespaces is available at https://api.cloudflare.com/client/v4/accounts/$YOUR_ACCOUNT_ID/workers/durable_objects/namespaces
-The API for listing objects in a namespace is available at https://api.cloudflare.com/client/v4/accounts/$YOUR_ACCOUNT_ID/workers/durable_objects/namespaces/$NAMESPACE_ID/objects
+The API for listing namespaces is available [here](https://api.cloudflare.com/client/v4/accounts/$YOUR_ACCOUNT_ID/workers/durable_objects/namespaces).
+The API for listing objects in a namespace is available [here](https://api.cloudflare.com/client/v4/accounts/$YOUR_ACCOUNT_ID/workers/durable_objects/namespaces/$NAMESPACE_ID/objects).
```

Comment:
these shouldn't be hyperlnks, they aren't usefully clickable. The URL itself is the information we're trying to convey, and would need to be populated with account ID and namespace ID as appropriate

Reply:
@koeninger thank you for catching that, I just saw the account ID part.